src/v2/modules/contacts/routes.js

```diff
@@ -28,3 +31,22 @@ exports.getDocuments = {
     }
   }
 };
+
+exports.postContact = {
+  method: 'POST',
+  path: '/crm/2.0/contacts',
+  handler: (request, h) => entityHandlers.createEntity(request, h, 'contact'),
+  options: {
+    description: 'Creates a new contact',
+    validate: {
+      payload: {
+        salutation: Joi.string().optional(),
+        firstName: Joi.string().optional(),
+        initials: Joi.string().optional(),
+        lastName: Joi.string().required(),
+        middleName: Joi.string().optional(),
+        isTest: Joi.boolean().optional().default(false)
```

Comment:
There are some common validators available in src/v2/lib/validators for fields such as isTest which are shared amongst many entities